DIPs/DIP1NNN-DK.md

```diff
@@ -0,0 +1,696 @@
+# @system variables
+
+| Field           | Value                                                           |
+|-----------------|-----------------------------------------------------------------|
+| DIP:            | (number/id -- assigned by DIP Manager)                          |
+| Review Count:   | 0                                                               |
+| Author:         | Dennis Korpel dkorpel@gmail.com                                 |
+| Implementation: |                                                                 |
+| Status:         |                                                                 |
+
+## Abstract
+When the `@system` attribute is attached to variables or fields, it means they cannot be written to in `@safe` functions.
+If the variable has an unsafe type, it also cannot be read in `@safe` code.
```

Comment:
What is an "unsafe type"?

Reply:
Read down a bit, but its there. Basically anything that is a pointer.

https://github.com/dlang/DIPs/pull/179/files#diff-62f04e927443e2ad2cc392bb9f890dc1R223

Reply:
In that case I think there should be at least a link to what the DIP means. It's akin to introducing an acronym without defining it then defining it hundreds of lines later.

Reply:
Good point.